### DISCUSSION TITLE
Rounded notification image

### DISCUSSION THREAD

**Discussion by OBLIXi-ON** [ErikReider/SwayNotificationCenter#379] [Q&A] [answered]
![Uploading IMG_20240202_164124__01.jpg…]()
Is there any way to make notification image have rounded corners?

**Comment by ErikReider** [comment, marked as answer]
The latest changes ship with a round notification image by default. The next stable release will include those changes

**Reply by OBLIXi-ON** [reply]
So I'll need to install the git version?

**Reply by ErikReider** [reply]
Yep :)
